package.json

```diff
@@ -20,33 +21,40 @@
   },
   "homepage": "",
   "dependencies": {
-    "@babel/polyfill": "^7.7.0",
+    "@types/pg": "^7.11.2",
+    "@types/supertest": "^2.0.8",
     "body-parser": "^1.19.0",
     "chai": "^4.2.0",
     "cors": "^2.8.4",
+    "custom-env": "^1.0.2",
```

Comment:
this too should be under `devDependency`,read this for more info https://medium.com/@stalonadsl948/dependencies-vs-devdependencies-926e096a3dee